src/main/java/me/timmy109/betterreports/commands/AdminCommand.java

```diff
@@ -42,53 +43,46 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		List<String> adminHelp = ArrayUtils.getAdminHelpList();
 		List<String> playerHelp = ArrayUtils.getPlayerHelpList();
 
-
 		if (args.length == 0) {
 			if (sender.hasPermission("betterreports.admin")) {
-				for (String s : adminHelp) {
-					sender.sendMessage(Common.color(s));
-				}
+				adminHelp.forEach(s -> sender.sendMessage(Common.color(s)));
 				return true;
 			}
 
-			for (String s : playerHelp) {
-				sender.sendMessage(Common.color(s));
-			}
+			playerHelp.forEach(s -> sender.sendMessage(Common.color(s)));
+			return true;
+		}
+
+		if (args.length > 1) {
+			sender.sendMessage(Common.color("&cUnknown Command"));
```

Comment:
Unknown command here is duplicated

Reply:
Not really, unless you want users to run `/betterreports reload stuff` and the command is still valid?

Reply:
See below, it's a duplicate of below.